llvm/lib/CGData/CodeGenData.cpp

```diff
@@ -215,6 +224,76 @@ void warn(Error E, StringRef Whence) {
   }
 }
 
+static std::string getPath(StringRef Dir, unsigned Task) {
+  return (Dir + "/" + llvm::Twine(Task) + ".saved_copy.bc").str();
```

Comment:
Will this work on windows? Can we use `llvm::sys::path`?

Reply:
> Will this work on windows? Can we use `llvm::sys::path`?

Thanks for the catch!